blocks/header/header.js

```diff
@@ -1,148 +1,241 @@
-        drop.setAttribute('tabindex', 0);
-        drop.addEventListener('focus', focusNavSection);
-      }
-    });
-  } else {
-    navDrops.forEach((drop) => {
-      drop.removeAttribute('role');
-      drop.removeAttribute('tabindex');
-      drop.removeEventListener('focus', focusNavSection);
-    });
-  }
-  // enable menu collapse on escape keypress
-  if (!expanded || isDesktop.matches) {
-    // collapse menu on escape press
-    window.addEventListener('keydown', closeOnEscape);
-  } else {
-    window.removeEventListener('keydown', closeOnEscape);
-  }
-}
-
 /**
  * decorates the header, mainly the nav
  * @param {Element} block The header block element
  */
 export default async function decorate(block) {
   // load nav as fragment
   const navMeta = getMetadata('nav');
-  const navPath = navMeta ? new URL(navMeta).pathname : '/nav';
+  // TODO: meta data in the document is not working
+  const navPath = navMeta ? new URL(navMeta).pathname : '/draft/vivesing/nav';
   const fragment = await loadFragment(navPath);
 
-  // decorate nav DOM
-  const nav = document.createElement('nav');
-  nav.id = 'nav';
-  while (fragment.firstElementChild) nav.append(fragment.firstElementChild);
+  // Global navigator  starts here
+  const globalNavigator = document.createElement('div');
+  globalNavigator.className = 'global-navigator';
+  const containerDiv = document.createElement('div');
+  containerDiv.className = 'container';
+  globalNavigator.append(containerDiv);
+  const navigationItems = fragment.querySelectorAll('.section.global-navigator li');
 
-  const classes = ['brand', 'sections', 'tools'];
-  classes.forEach((c, i) => {
-    const section = nav.children[i];
-    if (section) section.classList.add(`nav-${c}`);
+  navigationItems.forEach((singleNavigation) => {
+    const singleItemDiv = document.createElement('div');
+    singleItemDiv.className = 'global-section-item';
+    singleItemDiv.appendChild(singleNavigation.firstChild);
+    containerDiv.appendChild(singleItemDiv);
   });
+  block.append(globalNavigator);
+
+  // Top bar section starts here
+  const topBarSection = document.createElement('div');
+  topBarSection.className = 'top-bar';
+  const topBarContainerDiv = document.createElement('div');
+  topBarContainerDiv.className = 'container';
+  topBarContainerDiv.id = 'top-bar';
+  topBarSection.append(topBarContainerDiv);
+  const row1Div = document.createElement('div');
+  row1Div.className = 'row1';
+  topBarContainerDiv.appendChild(row1Div);
+
+  const hamburgerIconDiv = document.createElement('div');
+  hamburgerIconDiv.className = 'hamburger-menu';
+  hamburgerIconDiv.id = 'hamburger-menu-icon';
+  hamburgerIconDiv.innerHTML = `
+    <div></div>
+    <div></div>
+    <div></div>
+  `;
+
+  const logoDiv = document.createElement('div');
+  logoDiv.className = 'logo';
+  const logoImage = document.createElement('img');
+  logoImage.src = '../../icons/icicilogo.svg';
+  logoImage.alt = 'ICICI Logo';
```

Comment:
can we avoid hardcoding. `alt`, `icon` in the code? It should come from the doc if possible

Reply:
Okay sure I can keep these details in the doc itself.

Reply:
done